### ISSUE TITLE
Failed to Compile from Archlinux AUR (rkvm and rkvm-git)

### ISSUE THREAD

**Issue by g-h-97** [htrefil/rkvm#66]
It fails to compile with below error

```
          /usr/bin/ld: /home/user/.cache/yay/rkvm/src/rkvm-0.6.0/target/release/deps/rkvm_server-c049e940c6c16490: hidden symbol `GFp_ia32cap_P' isn't defined
          /usr/bin/ld: final link failed: bad value
          collect2: error: ld returned 1 exit status

  = note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-link-lib)

error: could not compile `rkvm-server` (bin "rkvm-server") due to 1 previous error
```

**Comment by g-h-97** [comment]
Solved by adding options=('!lto') to PKGBUILD. Just make sure to rebuild from scratch.

But being able to compile with LTO would be great. I'll keep the issue open for now.

**Comment by htrefil** [comment]
Fixed in 0.6.1.
